### PR TITLE
create the rents controller

### PR DESCRIPTION
**Pull request by cfbarros**

### CODE REVIEW

File: app/controllers/rents_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  @rent.bike = @bike
+  if @rent.save
+    redirect_to rent_path(@rent)
+  else
+    render :new
+  end
+end
+
+def show
+@rent = Rent.find(rent_params)
```
Comment (tpdoyle87) [inline app/controllers/rents_controller.rb]: the method is rent_param

File: app/controllers/rents_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class RentsController < ApplicationController
+
+def new
+  @rent = Rent.new
+end
+
+def create
+  @bike = Bike.find(params[:bike_id])
+  @rent = Rent.new(rent_params)
```
Comment (tpdoyle87) [inline app/controllers/rents_controller.rb]: the method is rent_param

Comment (cfbarros) [reply app/controllers/rents_controller.rb]: I've just changed it and pulled again.

**Comment by tpdoyle87** [comment]
looks good, i think you should merge it

**Pull request merged**